psbt_dump.py

```diff
@@ -416,6 +498,17 @@ def dump(psbt, hex_output, bin_output, testnet, base64, show_addrs):
                         # XXX decode implied p2sh addresses here too?
                 except:
                     print("(unable to parse POSSIBLE multisig redeem script)")
+    print()
+    if version:
+        if version == 0 and v2_fields:
+            print("INVALID: PSBT v0 requires exclusion of PSBT v2 fields.")
```

Comment:
unreachable

Reply:
changed `if version:` to `if version is not None`

what do you mean by unreachable ?